easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2-intel-2020a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'SCons'
+
+name = 'libobjcryst'
+version = '2017.2.2'
+
+homepage = 'https://github.com/diffpy/libobjcryst'
+description = "ObjCryst++ is Object-Oriented Crystallographic Library for C++"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+source_urls = ['https://github.com/diffpy/libobjcryst/releases/download/v%(version)s/']
+sources = ['libobjcryst-%(version)s.tar.gz']
+patches = [('libobjcryst-2017.2.2_sconscript_local.patch', 1)]
```

Comment:
EasyBuild automatically determines the value to pass to `patch -p` based on the files being patched, so you can simplify this to just:

```python
patches = ['libobjcryst-2017.2.2_sconscript_local.patch']

Reply:
Well, this didn't work because of a still-open issue that affects patches that create new files patch (sorry, forgot the issue number)

Reply:
You can dance around that by editing the patch so that the part that patches `libobjcryst-2017.2.3/SConstruct` is included first in the patch file.

Reply:
At least that was the first time around when I was only adding SConscript.local, now it works because the diff touches an exisitng file, at least that was what I understood from the issue discussion. ("Could not determine patch level for...")